core/src/main/scala/kafka/network/RequestConvertToJson.scala

```diff
@@ -194,6 +199,11 @@ object RequestConvertToJson {
       case res: UpdateMetadataResponse => UpdateMetadataResponseDataJsonConverter.write(res.data, version)
       case res: VoteResponse => VoteResponseDataJsonConverter.write(res.data, version)
       case res: WriteTxnMarkersResponse => WriteTxnMarkersResponseDataJsonConverter.write(res.data, version)
+      case res: InitializeShareGroupStateResponse => InitializeShareGroupStateResponseDataJsonConverter.write(res.data, version)
+      case res: ReadShareGroupStateResponse => ReadShareGroupStateResponseDataJsonConverter.write(res.data, version)
+      case res: WriteShareGroupStateResponse => WriteShareGroupStateResponseDataJsonConverter.write(res.data, version)
+      case res: DeleteShareGroupStateResponse => DeleteShareGroupStateResponseDataJsonConverter.write(res.data, version)
+      case res: ReadShareGroupOffsetsStateResponse => ReadShareGroupOffsetsStateResponseDataJsonConverter.write(res.data, version)
```

Comment:
I think this class follows order by name, so can we please add them respectively.

Reply:
Ok - this order is by KIP though

Reply:
Yeah, I only find this class following the order always. So we should try to maintain that.